ui/src/routes/community/+page.svelte

```diff
@@ -431,4 +432,6 @@
             </div>
         </div>
     </DisappearingContent>
+
+    <Footer color="pink"></Footer>
```

Comment:
The footer should be inside of the `DisappearingContent` section, otherwise the footer content can clash with the main menu / logo.